xcode.md

```diff
@@ -33,6 +33,7 @@ Got access to old versions of Apple installers you know are legit? Submit some h
 
 | Version                           | SHA1 Checksum                              | Filename                          |
 | --------------------------------- | ------------------------------------------ | --------------------------------- |
+| [Xcode 9.3 beta (10.12-10.13)][9.3b1] (9Q98q) | `3d1f4918810b173ff205b64a2006bb163d9cf5b1` | Xcode_9.3_beta.xip <!-- a3cf8fecf9c5a664cc8039914b51a4962546f1207e2c1514f219358167a03f4f -->
```

Comment:
It’s only 10.13 now.

Reply:
Aah... my bad. Should I correct it and send you another pull request?

Reply:
Yeah, just push another commit with the fix to your branch, and we should be able to just do it from this pull request. I’ll merge it once it looks OK.

Reply:
Well, I've done it - take a look.